packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -165,12 +170,12 @@ export class DebugSession implements Disposable {
       .then(() => disposeAll(this.disposables));
   }
 
-  public async startJSDebugSession(metro: Metro) {
+  public async startJSDebugSession(metro: Metro, extraConfiguration?: DebugExtraConfiguration) {
     if (this.jsDebugSession) {
       await this.restart();
     }
 
-    const websocketAddress = await metro.getDebuggerURL();
+    let websocketAddress = extraConfiguration?.websocketAddress || (await metro.getDebuggerURL());
```

Comment:
```suggestion
    const websocketAddress = extraConfiguration?.websocketAddress || (await metro.getDebuggerURL());
```
This is never overwritten, right?